server/routes/api/v1/auth.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import AuthControllers from '../../../controllers/AuthControllers';
```

Comment:
Why do you have a controller called AuthController?

Reply:
I updated the earlier created userController created to AuthControllers

Reply:
The controller should still be called UserController (capitalised) it's the route file for signup that should be called 'auth'

Reply:
What I remember @KvNGCzA  earlier said was signin and signup belongs to both admin and all types of users, therefore they can have a route named auth and a controller also named AuthController, while UserController and user route is for what users can do but admin cannot do.

Reply:
Treated accordingly